<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audioToolUI</class>
 <widget class="QWidget" name="audioToolUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://Service_Tools/Service_Tools/tools_menu.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>10</y>
      <width>616</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">Service Panel/Tools/Audio setup</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>285</y>
      <width>291</width>
      <height>116</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,50);
background-image: url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="volMButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>81</width>
       <height>66</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pulsanti/Pulsanti/scroll_left.png);
background-color: rgb(0, 0, 0,0);
background-image: url(:/transparent.png);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="volPButton">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>40</y>
       <width>81</width>
       <height>66</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pulsanti/Pulsanti/scroll_right.png);
background-image: url(:/transparent.png);
background-color: rgb(0, 0, 0,0);</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>276</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0,0);
color: rgb(255, 255, 85);</string>
     </property>
     <property name="text">
      <string notr="true">VOLUME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="volume">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>45</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0,0);</string>
     </property>
     <property name="text">
      <string notr="true">100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>100</y>
      <width>291</width>
      <height>166</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,50);
background-image: url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>276</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0,0);
color: rgb(255, 255, 85);</string>
     </property>
     <property name="text">
      <string notr="true">TEST MESSAGE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="testButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>45</y>
       <width>136</width>
       <height>116</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(://Service_Tools/Service_Tools/audioPix.png);
background-image: url(:/transparent.png);
background-color: rgb(0, 0, 0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="msgNum">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>115</y>
       <width>121</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string notr="true">#1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,50);
background-image: url(:/transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>276</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bitstream Charter</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="accessibleDescription">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0,0);
color: rgb(255, 255, 85);</string>
     </property>
     <property name="text">
      <string notr="true">ENABLE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="enabButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>80</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pulsanti/Pulsanti/but_on.png);
background-image: url(:/transparent.png);
background-color: rgb(0, 0, 0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="notDetectedLabel">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>440</y>
      <width>641</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0,0);
background-image: url(:/transparent.png);
color: rgb(255, 117, 82);</string>
    </property>
    <property name="text">
     <string notr="true">AUDIO DEVICE NOT DETECTED !!!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="notDetectedFrame">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>100</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/transparent.png);
background-color: rgb(173, 173, 173,150);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
